test/test.oauth2.ts

```diff
@@ -1149,6 +1150,36 @@ describe('oauth2', () => {
           done();
         });
       });
+
```

Comment:
Can you try to structure your tests as follows (whenever you can):
```
// Setup test block. You define the variables and stubs here.
// Add empty line.
// Run test block. You call the API you want to test.
// Add empty line.
// Assert test behavior block.
```

Sometimes you can group the last 2 blocks.

This will make it easy to know what you are actually testing.